packages/form/src/index.js

```diff
@@ -1,3 +1,5 @@
 export * from './components';
 
-export * from './hooks/use-field';
+export * from './hooks';
+
+export { STORE_NAME } from './data';
```

Comment:
I'm not sure we should be exporting this - see my comments above on how to deal with that potentially.